smarts/core/vehicle_index.py

```diff
@@ -526,8 +526,8 @@ def build_social_vehicle(
 
         return vehicle
 
-    def sensor_state_for_vehicle_id(self, vehicle_id):
-        return self._sensor_states[vehicle_id]
+    def sensor_states(self):
+        return self._sensor_states
```

Comment:
I think this is fine. A dictionary-like object is a reasonable interface here.

Reply:
Can we keep the old? It's consistent with the other method signatures (*_for_vehicle_id).

Reply:
All I need is a list of vehicles with sensors, is it not possible to maintain such a list outside of vehicle index?

Reply:
Ok, I have kept sensor_state_for_vehicle_id().

Instead, I have created sensor_states_items() to return a new view of the sensor states dictionary's items, similar to how it is done for vehicleitems().

Reply:
> Can we keep the old? It's consistent with the other method signatures (*_for_vehicle_id). Returning the dictionary allows for manipulation of the reference data in an uncontrolled manner.

I had missed the reference manipulation. I would see it as fine if it were exposed as a copy. @aurorahcx 's `items()` seems to work fine now.